wiki/Beatmap/Category/id.md

```diff
@@ -1,59 +1,51 @@
+Pada masanya dahulu, beatmap-beatmap Approved tidak memengaruhi total skor yang dimiliki oleh masing-masing pemain. Setelah osu! bertransisi dari sistem *score ranking* menuju ke sistem [*performance point ranking*](/wiki/Performance_points), status Approved kemudian dialihfungsikan untuk menandai beatmap-beatmap yang memiliki [*drain time*](/wiki/Gameplay/Drain_time) 6 menit atau lebih sebelum akhirnya perlahan dipensiunkan.
 
-Adapun beatmap-beatmap ini ditandai dengan simbol centang (![](/wiki/shared/status/approved.png)) yang dapat dilihat pada pojok kiri atas layar Song Selection.
+Adapun beatmap-beatmap ini ditandai dengan simbol centang (![](/wiki/shared/status/approved.png)) yang dapat dilihat pada pojok kiri atas layar song selection.
 
-### Loved
+### Kemunculan kategori Qualified
 
-*Untuk mengetahui lebih lanjut seputar latar belakang status Loved, harap kunjungi: [Sejarah Loved](/wiki/Beatmap/History_of_Loved)*
+Kategori [Qualified](#qualified) pertama kali diperkenalkan pada bulan Agustus 2014 saat [restrukturisasi BAT](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) berjalan untuk membuat beatmap yang di-ranked lebih kelihatan. Jika beatmap dalam kategori Qualified perlu didiskualifikasi dan dikembalikan ke Pending, hal tersebut tidak akan mempengaruhi statistik pemain, karena kategori tersebut sudah didesain dari awal untuk tidak mencatat skor. Kemunculan kategori ini disambut dengan baik karena pemain dapat mencoba dan menentukan apakah map tersebut sudah memenuhi ekspektasi atau belum.
+
+### Statistik beatmap unranked
 
-Loved merupakan status beatmap yang disematkan kepada beatmap-beatmap yang *dicintai (loved)* oleh anggota-anggota komunitas berdasarkan hasil jajak pendapat yang diselenggarakan oleh tim [Project Loved](/wiki/Project_Loved) terlepas dari apakah beatmap-beatmap tersebut memenuhi standar [Ranking Criteria](/wiki/Ranking_Criteria) ataupun tidak. Beatmap-beatmap pada kategori ini tidak memberikan [*performance point*](/wiki/Performance_points) apapun, namun memiliki *leaderboard* yang dapat sewaktu-waktu dihapus apabila beatmap-beatmap yang bersangkutan dicabut dari kategori Loved karena satu dan lain hal. Adapun beatmap-beatmap ini ditandai dengan simbol hati (![](/wiki/shared/status/loved.png)) yang dapat dilihat pada pojok kiri atas layar Song Selection.
+Sejak tanggal [7 Agustus 2020](https://osu.ppy.sh/home/changelog/stable40/20200807.3), beatmap dari kategori [Graveyard](#graveyard) dan [Work in Progress dan Pending](#work-in-progress-and-pending) juga berkontribusi terhadap jumlah main dan skor total dari pemain. Beatmap dengan kategori tersebut juga memiliki statistik bermainnya sendiri (jumlah main, tingkat kesuksesan, dan letak-letak kegagalan) yang sebelumnya hanya ditunjukkan untuk beatmap [Qualified](#qualified) dan [Ranked](#ranked).
```

Comment:
`Per tanggal [7 Agustus 2020](https://osu.ppy.sh/home/changelog/stable40/20200807.3), beatmap-beatmap yang berstatus [Graveyard](#graveyard) serta [Work in Progress dan Pending](#work-in-progress-and-pending) juga berkontribusi terhadap jumlah permainan dan skor total para pemain. Beatmap-beatmap yang berada pada kedua kategori tersebut juga kini memiliki statistik permainannya tersendiri (yang meliputi antara lain jumlah permainan, tingkat keberhasilan, dan titik-titik kegagalan) yang sebelumnya hanya dapat dilihat pada beatmap-beatmap [Qualified](#qualified) dan [Ranked](#ranked).`

![image](https://user-images.githubusercontent.com/36564236/147039355-26a39d4b-8354-43fd-bd8d-f1ee59ef1e68.png)